IP/src/main/java/Ui.java

```diff
@@ -0,0 +1,81 @@
+public class Ui {
+
+    private String lines = "____________________________________________________________";
+    private static String greeting = """
+                Hello! I'm a meerkat
+                What can I do for you?
+                """;
+    private static String bye = """
+                Goodnight, sleep tight, Hope I don't ever see you again!
+                """;
+
+    public void printList(TaskList taskList) {
+        System.out.println(lines);
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 1; i < taskList.getSize() + 1; i++) {
+            System.out.println(i + "." + taskList.getTaskList().get(i-1).toString());
+        }
+        System.out.println(lines);
+    }
+
+    public void printTaskCreationMessage(Task thisTask, TaskList taskList) {
+        System.out.println(lines + "\nGot it. I've added this task:\n  " + thisTask + "\nNow you have " + taskList.getSize() + " tasks in the list\n" + lines);
```

Comment:
Don't know the exact number of characters but this line could be indented by entering to next line.